homeassistant/components/aftership/strings.json

```diff
@@ -32,5 +32,34 @@
         }
       }
     }
+  },
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
```

Comment:
Probably copy-paste error. Please specify only the api key

Reply:
hehehe yes